### PR TITLE
LPS-156124

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related ticket:
https://issues.liferay.com/browse/LPS-156124

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
@carolmariaabb this PR contains commits from LPS-156125 also. They were already reviewed by you. Therefore, before forwarding this, let's wait to see if the other PR gets merged without changes.
Thx

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java [hunk 1]
```diff
@@ -628,10 +628,41 @@ private void _validateParameterObjectFieldId(
...
+					StringBundler.concat(
+						"Parameter object field ", parameterObjectFieldId,
+						" does not belong to object definition ",
+						objectDefinition.getName()));
+			}
+
+			if (Objects.equals(
+					objectField.getBusinessType(),
+					ObjectFieldConstants.BUSINESS_TYPE_RELATIONSHIP)) {
+
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java:660]: Shouldn't it be `!Objects.equals(..)` instead?

Comment (rodrigopaulino) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java]: @carolmariaabb, I moved this change to LPS-156125. If you want to check, you can see here in this same pull request, but I already sent it along with the other PR that contains Brian's requests.

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Review by carolmariaabb — APPROVED** [review]

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:test:sf

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:forward

**Pull request closed without merging**